src/main/java/org/transitopia/layers/Cycling.java

```diff
@@ -207,7 +209,24 @@ else if (feature.hasTag("highway") && !feature.hasTag("oneway", "yes") &&
                     newLine.setAttrWithMinzoom("routes", routeIdsString, MIN_ZOOM_ATTR);
                 }
             }
+        } else if (feature.isPoint()) {
+            if (feature.hasTag("amenity", "bicycle_parking")) {
+                features.point(LAYER_NAME)
+                    .setAttr("amenity", "bicycle_parking")
+                    .setAttr("name", feature.getTag("name"))
+                    .setAttr("operator", feature.getTag("operator"))
+                    .setAttr("indoor", feature.getTag("indoor"))
+                    .setAttr("access", feature.getTag("access")) // TODO: validate, rstrict to private/customers/members
+                    .setAttr("capacity", feature.getTag("capacity"))
+                    .setAttr("bicycle_parking", feature.getTag("bicycle_parking")) // TODO: validate and ignore if not a known value
+                    .setAttr("covered", feature.getTag("covered"))
+                    .setAttr("cyclestreets_id", feature.getTag("cyclestreets_id"))
+                    .setAttr("fee", feature.getTag("fee"))
+                    .setMinZoom(MIN_ZOOM_DETAILS);
+            }
+            // TODO: amenity=kick-scooter_parking
```

Comment:
kick scooter parking is barely used at all - skip it.

https://overpass-turbo.eu/?template=key-value&key=amenity&value=kick-scooter_parking